dao/src/main/scala/za/co/absa/enceladus/dao/auth/MenasCredentials.scala

```diff
@@ -55,9 +55,9 @@ object MenasKerberosCredentials {
     * @return An instance of Menas Credentials.
     */
   def fromFile(path: String)(implicit spark: SparkSession): MenasKerberosCredentials = {
-    val fsUtils = new FileSystemVersionUtils(spark.sparkContext.hadoopConfiguration)
+    val fsUtils = new HdfsUtils(spark.sparkContext.hadoopConfiguration)
 
-    val localKeyTabPath = fsUtils.getLocalPathToFile(path)
+    val localKeyTabPath = fsUtils.getLocalPathToFileOrCopyToLocal(path)
```

Comment:
The original logic is to load from local path and fallback on the same path to HDFS. Do we want to have the option to fallback to a menas-credentials file on S3?

Reply:
Are you asking if we want the ability to read menas-credentials file from S3 in general? Answer is yes

Or in some special case?

Reply:
Thanks, yes, the former (`ability to read menas-credentials file from S3 in general`).

Reply:
I think fallback is not necessary